about.html

```diff
@@ -0,0 +1,122 @@
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </section>
+      <section class="nav-desktop">
+        <a href="#" id="about-open-btn">
+          <i class="bi bi-list"></i>
+        </a>
+        <a href="#" class="logo">SUNSHINE ROCK<br>MUSIC FEST</a>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="#logo-section">Logo</a></li>
+          <li><a href="#past-section">Past Events</a></li>
+          <li><a href="#sponsor-section">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li class="square"><a href="#">CC Campaign</a></li>
+        </ul>
+      </section>
+    </nav>
+    <nav class="mobile-menu-off" id="menu-mobile-about">
+      <i class="bi bi-x-lg" id="about-close-btn"></i>
+      <ul>
+        <li><a id="home-btn" href="./index.html">Home</a></li>
+        <li><a id="logo-btn" href="#logo-section">Logo</a></li>
+        <li><a id="past-btn" href="#past-section">Past Events</a></li>
+        <li><a id="sponsor-about-btn" href="#sponsor-section">Sponsor</a></li>
+        <li><a id="news-about-btn" href="#">News</a></li>
+        <li class="square"><a id="camp-about-btn" href="#">CC Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+    <main>
+      <section id="intro" class="intro-section intro-about">
+        <div class="path">
+        <ul>
+          <li><a href="./index.html"><i class="bi bi-house-fill"></i></a></li>
+          <li><i class="bi bi-chevron-right greater"></i></li>
+          <li><a href="#">About</a></li>
+        </ul>
+      </div>
+        <h1>SUNSHINE ROCK<br>MUSIC FEST 2022</h1>
+        <h2>"Hello! Sunny world"</h2>
+        <div class="text-container">
+          <p>A joyful celebration believing in the value of openes and sharing, creating a positive change from people
+            over 80 countries is taking place in September in Argentina.
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsum aperiam rem eveniet saepe porro voluptatem
+            iure.</p>
+        </div>
+        <h4>Please contact us by mail for any further questions about<br>SUNSHINE ROCK MUSIC FEST 2022!</h4>
+        <a class="about-mail" href="#">sunshinerock@musicargentina.com</a>
+      </section>
+      <section id="logo-section" class="logo-section">
+        <h2>Sunshine Rock<br>
+        Music Festival 2022 Logo</h2>
+        <div class="horizontal-bar"></div>
+        <h3>The Logo of Sunshine Rock Music Fest 2022 was decided throug a competition from 7 April to 10 May.</h3>
+        <div class="logo-container">
+          <h4>Sunshine Rock<br>
+          Music Fest</h4>
+        </div>
+      </section>
+      <section id="past-section" class="past-section">
+        <h2>See the past<br>Sunshine Rock Music Fest!</h2>
+        <div class="horizontal-bar"></div>
+        <h3>Take a look to the last two Sunshine Rock Music Fest which took place in London and Tokio.</h3>
+        <div class="events-container">
```

Comment:
- [ ] Please consider using two images for this section, this will make your design closer to the provided one

![Screenshot_20220519_202227](https://user-images.githubusercontent.com/25227022/169386112-47c348d7-fceb-430d-b2f5-9f533872e784.png)